### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by sohilpandya**
Minor bug fix related to issue #314 & #317 
Please review after merging #313

### CODE REVIEW

File: server/plugins/modules.js [hunk 1]
```diff
@@ -109,23 +109,17 @@ exports.register = (server, options, next) => {
...
-                    if (!module_id) {
-                        return reply(new Error('module_id must be defined'));
-                    }
+                const { module_id, user_id } = request.query;
 
-                    getStudentHistory(pool, user_id, module_id, (error, history) => {
-                        const verdict = error || history;
-                        reply(verdict);
-                    });
+                if (!module_id) {
```
Comment (Danwhy) [inline server/plugins/modules.js]: Did we need to check if user_id exists here too?

Comment (shouston3) [inline server/plugins/modules.js]: @Danwhy nice spot changing

**Comment by shouston3** [comment]
@Danwhy All good to go

**Comment by sohilpandya** [comment]
@Danwhy Should be good to go now. 👍

**Comment by iteles** [comment]
:+1:

**Comment by sohilpandya** [comment]
Thanks @Danwhy

**Pull request merged**